### DISCUSSION TITLE
Discussion or Issue?

### DISCUSSION THREAD

**Discussion by snipe** [snipe/snipe-it#13260] [General]
Hi all,

I wanted to throw up this pinned discussion so folks can better determine when something should be a Discussion vs an Issue. If you're running into an error (as in, there is an actual error message) this is most likely an Issue, not a Discussion. If you're curious about how other people are doing things, that's more likely a Discussion. 

While the devs do check Discussions, they don't check them as diligently as they do Issues, so if you're posting a bug here, it's likely to take longer to be seen. Discussions are a place for the community to share notes about how they solve problems, or to discuss *potential* features and workflows. 

Hope this helps! 
Até breve!

**Comment by samuelgodson55** [comment]
Please we need a bulk checkin features for accessories. I work for a rental company and we track items using quantities.

Say 200 paganilis and wedges was checked out to a client. It will be unreasonable to start checking them in one after the other on return.

**Comment by rschugh** [comment]
We want to add a indent option where user can indent any asset on any item... and manager will approve and admin assigned asset further on it... how is it possible

**Reply by snipe** [reply]
Please don't post unrelated comments on this. It's an informational thread only.

Your issue is a duplicate of https://github.com/grokability/snipe-it/issues/8476
